custom_components/hypervolt_charger/hypervolt_api_client.py

```diff
@@ -75,12 +75,10 @@ async def login(self, session: aiohttp.ClientSession) -> str:
             session.headers["user-agent"] = self.get_user_agent()
 
             async with session.post(
-                "https://auth.hypervolt.co.uk/oauth/token",
+                "https://kc.hypervolt.co.uk/realms/retail-customers/protocol/openid-connect/token",
```

Comment:
This raises
```'Cannot connect to host kc.hypervolt.co.uk:443 ssl:default [Name or service not known]'```

If I `dig kc.hypervolt.co.uk` I don't get a response.

Reply:
I tried:
`kb.hypervolt.co.uk` -> gives 403
`kc.prod.hypervolt.co.uk` -> gives 400
🤷